sat/main.py

```diff
@@ -119,7 +120,11 @@ def main():
                          args.command, args.command)
             sys.exit(1)
 
-        subcommand(args)
+        try:
+            subcommand(args)
+        except InvalidGrantError as err:
```

Comment:
`err` is unused, so drop it

```suggestion
        except InvalidGrantError:
```